src/site/content/en/fast/cookie-notice-best-practices/index.md

```diff
@@ -0,0 +1,457 @@
+have a greater impact on performance than a cookie notice that you build
+yourself. This is not a problem unique to cookie notices—but rather the nature
+of third-party scripts in general.
+
+
+### Best practices
+
+The best practices in this section focus on third-party cookie notices. Some,
+but not all, of these best practices will also be applicable to first-party
+cookie notices.
+
+
+#### Load cookie notices scripts asynchronously
+
+Cookie notice scripts should be loaded asynchronously. To do this, add the
+[`async`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-async)
+attribute to the script tag.
+
+```html
+<script src="https://cookie-notice.com/script.js" async>
+```
+
+Scripts that are not asynchronous block the browser parser. This delays page
+load and LCP. For more information, see [Efficiently load third-party
+JavaScript](https://web.dev/efficiently-load-third-party-javascript/).
+
+{% Aside %}
+If you must use synchronous scripts (for example, some cookie notices rely on
+synchronous scripts to implement cookie blocking) you should make sure that this
+request loads as quickly as possible. One way to do this is to [use resource
+hints](https://web.dev/preconnect-and-dns-prefetch/).
+{% endAside %}
+
+
+#### Load cookie notice scripts directly
+
+Cookie notice scripts should be loaded "directly" by placing the script tag in
+the HTML of the main document—rather than loaded by a tag manager or other
+script. Using a tag manager or secondary script to inject the cookie notice
+script delays the loading of the cookie notice script: it obscures the script
+from the browser's lookahead parser and it prevents the script from loading
+before JavaScript execution.
+
+
+#### Establish an early connection with the cookie notice origin
+
+All sites that load their cookie notice scripts from a third-party location
+should use either the `dns-prefetch` or `preconnect` resource hints to help
+establish an early connection with the origin that hosts cookie notice
+resources. For more information, see [Establish network connections early to
+improve perceived page speed](https://web.dev/preconnect-and-dns-prefetch/).
+
+```html
+<link rel="preconnect" href="https://cdn.cookie-notice.com/">
+```
+
+{% Aside %}
+It is common for cookie notices to load resources from multiple origins—
+for example, loading resources from both
+`www.cookie-notice.com` and
+`cdn.cookie-notice.com`. Separate origins require separate connections and
+therefore separate resource hints.
+{% endAside %}
+
+
+#### Preload cookie notices as appropriate
+
+Some sites would benefit from using the
+[`preload`](https://developer.mozilla.org/en-US/docs/Web/HTML/Preloading_content)
+resource hint to load their cookie notice script. The `preload` resource hint
+informs the browser to initiate an early request for the specified resource.
+
+```html
+<link rel="prefetch" href="https://www.cookie-notice.com/cookie-script.js">
```

Comment:
Since the surrounding paragraphs are talking about preloading, perhaps that's what was meant here as well?

```diff
- <link rel="prefetch" href="https://www.cookie-notice.com/cookie-script.js">
+ <link rel="preload" as="script" href="https://www.cookie-notice.com/cookie-script.js">
```